source/examples/xgboost-dask-databricks/notebook.ipynb

```diff
@@ -0,0 +1,2163 @@
+{
```

Comment:
<p>I'm curious why this is in a function, and then we just call the function once? If we only need it one time let's just run this in a cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>